src/test/resources/features/Product-page-tests.feature

```diff
@@ -1,9 +1,14 @@
 Feature: Test product page elements and functionality
 
   // Author: Priyanka
-
   Scenario: Validate click on Search Option
     Given Visit The Shop Website
     When Click on the Shop tab
     And Search the product
+    Then Product should be displayed
+
+  Scenario: Validate click on Men's Clothing Tab
+    Given Visit The Shop Website
+    When Click on the Shop tab
```

Comment:
This step should be in TA-68?

Reply:
But these steps are also included in product page.